microsoft/helium
# Aka/actor query by name

Closes #208 

**What this PR does / why we need it**:
Adds the ability to search for an actor by name.

**Special notes for your reviewer**:

To test manually:
1. Start server (using npm)
2. Get all actors by sending a GET request to `api/actors` endpoint (using Postman or browser, etc.)
3. Pick a random actor name from the JSON results
4. Add the actor name to the GET request, send GET to modified endpoint of `api/actors?q=<actor name>`
5. Only actors whose name contains the string you added as a query should appear in the results.

With docker:
1. Run `docker-compose build`
2. Run `docker-compose up`
3. Integration tests should run automatically and demonstrate that a new actor was successfully posted and retrieved using the name filter.

**If applicable**:
